recipes/scrapbook/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.3.1" %}
+
+package:
+  name: scrapbook
```

Comment:
Shouldn't this have the same name as the pypi package?

Reply:
I was a bit hesitant. It seems to me the pypi package is named nteract-scrapbook and not scrapbook because scrapbook was already taken. Other projects from the nteract organization do not have this prefix. I favoured the simplicity by dropping the prefix, but if you think it's more consistent to keep it I can leave it. :)

Reply:
I would like to keep this in line with the PyPI package.